src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -225,4 +227,52 @@ private function initCache($cache, ContainerBuilder $container, Loader\XmlFileLo
 
         $loader->load('cache.xml');
     }
+
+    /**
+     * Initializes list builder.
+     *
+     * @param ContainerBuilder $container
+     * @param Loader\XmlFileLoader $loader
+     */
+    private function initListBuilder(ContainerBuilder $container, Loader\XmlFileLoader $loader)
+    {
+        $loader->load('list_builder.xml');
+
+        $metadataPaths = $this->getBundleMappingPaths($container->getParameter('kernel.bundles'), 'list-builder');
+        $fileLocator = $container->getDefinition('sulu_core.list_builder.metadata.file_locator');
+        $fileLocator->replaceArgument(0, $metadataPaths);
+    }
+
+    /**
+     * Returns list of bundle config paths.
+     *
+     * @param string[] $bundles
+     * @param string $dir
+     *
+     * @return array
+     */
+    private function getBundleMappingPaths($bundles, $dir)
+    {
+        $metadataPaths = [];
+        foreach ($bundles as $bundle) {
+            $refl = new \ReflectionClass($bundle);
```

Comment:
There would also be the `getPath` method of the Symfony `Bundle` class, which does exactly the same. Can you maybe change that in this PR, will be merged back to master on the release, since it doesn't make any difference.